src/components/metrics.tsx

```diff
@@ -71,6 +71,9 @@ import { colors, Error, QueryBrowser } from './query-browser';
 import TablePagination from './table-pagination';
 import { PrometheusAPIError, RootState } from './types';
 
+import { CSVLink } from 'react-csv';
+import ExternalLinkSquareAltIcon from '@patternfly/react-icons/dist/esm/icons/external-link-square-alt-icon';
```

Comment:
Shouldn't we be using the "Download" or "File Download' icon rather than the external link icon? (See https://preview.patternfly.org/icons/).

FYI @fkargbo

Reply:
Instead of this icon. The option should be included as part of the kebab menu as proposed on the mockups

Reply:
Good catch @kyoto , that is the "external link" icon. PatternFly has an icon for "Download" and for "Export". We should probably align the icon that matches the text as per the Option 1 diagram if we're handling multiple file formats, i.e. CSV, PNG, PDF, etc... 

Alternatively, we could use just the text as shown in Option 3 if we're only exporting a single file format. Here's a link to the Google Design doc. https://docs.google.com/document/d/1OgU41FCRO6PGisZU66ky00KYrL9HhTObILlPQkHHonI/edit

Let me know if I can help with anything. Thanks!

Reply:
Ah, yes, that makes sense. In the latest commit, I removed the icon and followed the mock-ups for Option 2. The 'Export as CSV' option is now a DropdownItem in the QueryKebab menu. 

 Also, I linked mock-up JIRAs ([OU-235](https://issues.redhat.com/browse/OU-235) and [OU-237](https://issues.redhat.com/browse/OU-237)) to the respective dev JIRAs ([OU-236](https://issues.redhat.com/browse/OU-236) and [OU-238](https://issues.redhat.com/browse/OU-238)).